content/en/docs/concepts/workloads/pods/sidecar-containers.md

```diff
@@ -74,10 +74,30 @@ If you define a Job that uses sidecar using Kubernetes-style init containers,
 the sidecar container in each Pod does not prevent the Job from completing after the
 main container has finished.
 
+{{% code_sample language="yaml" file="application/deployment-sidecar.yaml" %}}
+
+This feature is also useful for running Jobs with sidecars, as the sidecar
+container will not prevent the Job from completing after the main container
+has finished.
+
 Here's an example of a Job with two containers, one of which is a sidecar:
 
 {{% code_sample language="yaml" file="application/job/job-sidecar.yaml" %}}
 
+### Sidecar containers defined outside of `initContainers` field (legacy approach) {#sidecar-container-legacy-approach}
+
+Compared to regular defined within `initContainers`, you could 
+directly use multiple `.spec.containers` definitions for a Pod. If you do so, the
+kubelet starts the containers from the ordered `.spec.containers` list.
+
+Here's an example of a Deployment with two containers:
```

Comment:
I don't think the two examples really help clarify anything.
Does it hurt if we drop these two additional examples?

Reply:
I believe he added it to show a hands-on illustration. However, it could distract readers from the example describing the standard way to apply a side-car container.